### PR TITLE
deps: update outdated transitive deps

### PR DESCRIPTION
**Pull request by brendankenny**
fixes some security alert noise

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1140,20 +1140,21 @@ axe-core@3.3.0:
...
   dependencies:
-    follow-redirects "1.0.0"
+    follow-redirects "1.5.10"
+    is-buffer "^2.0.2"
 
 axios@^0.18.0:
-  version "0.18.0"
-  resolved "https://registry.yarnpkg.com/axios/-/axios-0.18.0.tgz#32d53e4851efdc0a11993b6cd000789d70c05102"
-  integrity sha1-MtU+SFHv3AoRmTts0AB4nXDAUQI=
+  version "0.18.1"
```
Comment (patrickhulce) [inline yarn.lock]: 😱  what no we can't use `0.18.1`!!! 

:)

File: yarn.lock [hunk 2]
```diff
@@ -2468,12 +2469,12 @@ debug@2.6.9, debug@^2.2.0, debug@^2.3.3, debug@^2.6.8:
   dependencies:
     ms "2.0.0"
 
-debug@^3.1.0, debug@^3.2.6:
-  version "3.2.6"
-  resolved "https://registry.yarnpkg.com/debug/-/debug-3.2.6.tgz#e83d17de16d8a7efb7717edbe5fb10135eee629b"
-  integrity sha512-mel+jf7nrtEl5Pn1Qx46zARXKDpBbvzezse7p7LqINmdoIk8PYP5SySaxEmYv6TZ0JyEKA1hsCId6DIhgITtWQ==
+debug@=3.1.0, debug@^3.1.0:
```
Comment (patrickhulce) [inline yarn.lock]: interesting that  follow-redirects dropped *down* to 3.1.0 of debug and can only use 3.1.0...

Comment (brendankenny) [reply yarn.lock]: > interesting that follow-redirects dropped down to 3.1.0 of debug and can only use 3.1.0...

oh yeah, really weird. I'm almost curious enough to dig into it, but not quite :)

Comment (brendankenny) [reply yarn.lock]: ah, the answer was not that interesting and makes sense:

["Fix to debug 3.1.0 for Node 4 compatibility."](https://github.com/follow-redirects/follow-redirects/commit/ff8a78a15c52e6d5b1e219a2b7de4ec54ca747b9)

(they updated to 3.2.6 and then to 4.something in releases in the months after that, but axios hasn't kept up)

**Pull request merged**